terraform/eks/agent.tf

```diff
@@ -0,0 +1,53 @@
+data "template_file" "cluster_role_file" {
```

Comment:
I would suggest to add some comments for this file to explain it is for container insights. Or you could rename the file to `CIAgent.tf`

Reply:
Oops, I thought CI stood for continuous integration...